### PR TITLE
add hyphenation support

### PR DESCRIPTION
**Pull request by MartinNowak**
- via --hyphenate argument to serve/generate-html

### CODE REVIEW

File: dub.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
 	],
 	"dependencies": {
   		"vibe-d": "~>0.7.22",
-		"libdparse": {"optional": true, "version": "~>0.2.0"}
+		"libdparse": {"optional": true, "version": "~>0.2.0"},
+		"hyphenate": "~>1.1.0"
```
Comment (MartinNowak) [inline dub.json]: It's just a single module and a pattern file, so I thought it wasn't worth the effort to handle `Have_hyphenate` everywhere (in particular b/c it's a command line option).

File: dub.selections.json [hunk 2]
```diff
@@ -4,9 +4,10 @@
 		"libevent": "2.0.1+2.0.16",
 		"libev": "5.0.0+4.04",
 		"openssl": "1.1.4+1.0.1g",
+		"libdparse": "0.2.1",
```
Comment (s-ludwig) [inline dub.selections.json]: Adding this effectively would make libdparse non-optional when buillding DDOX directly, but I guess we can count on libdparse enough to make it non-optional (I'll have a second look at #104 and merge it next, including the actual non-optional change there).

**Comment by s-ludwig** [comment]
LGTM!

File: dub.selections.json (hunk 2 above)
Comment (MartinNowak) [inline dub.selections.json]: Whoops, it's too easy to overlook this.
I should fix dub to at least always output a stable order.

**Comment by MartinNowak** [comment]
Can we tag a new release (0.12.1) for this, so I can submite my dlang.org hyphenate PR today :)?

**Comment by s-ludwig** [comment]
Makes, sense. Especially before pulling #104.

**Comment by s-ludwig** [comment]
http://code.dlang.org/packages/ddox/0.12.1

**Comment by MartinNowak** [comment]
> Makes sense. Especially before pulling #104.

Yes, the next one should be 0.13.0.

**Pull request merged**